astropy/io/misc/asdf/types.py

```diff
@@ -21,20 +22,34 @@ def __new__(mcls, name, bases, attrs):
         cls = super(AstropyTypeMeta, mcls).__new__(mcls, name, bases, attrs)
         # Classes using this metaclass are automatically added to the list of
         # astropy extensions
-        if cls.organization == 'stsci.edu' and cls.standard == 'asdf':
+        if cls.organization == 'astropy.org' and cls.standard == 'astropy':
+            _astropy_types.add(cls)
+        elif cls.organization == 'stsci.edu' and cls.standard == 'asdf':
             _astropy_asdf_types.add(cls)
 
         return cls
 
 
+@six.add_metaclass(AstropyTypeMeta)
+class AstropyType(CustomType):
+    """
+    This class represents types that have schemas that are defined by Astropy.
+
+    IMPORTANT: This parent class should **not** be used for types that also
+    have schemas that are defined by astropy.
+    """
+    organization = 'astropy.org'
+    standard = 'astropy'
+
+
 @six.add_metaclass(AstropyTypeMeta)
 class AstropyAsdfType(CustomType):
     """
     This class represents types that have schemas that are defined in the ASDF
     standard, but have tags that are implemented within astropy.
 
-    IMPORTANT: This parent class should **not** be used for types that also
-    have schemas that are defined by astropy.
+    IMPORTANT: This parent class should **not** be used for types that have
+    schemas that are defined by the ASDF standard.
```

Comment:
Oops, did the correction go in the wrong class? Just to make sure I have this straight. `AstropyAsdfType` is for types with schemas in asdf, right? If so, the comment was correct in this class and wrong in `AstropyType`. Or is it the other way around?

Reply:
Yes, you're right. Sorry about that.